README.md

```diff
@@ -150,7 +150,7 @@ My mother. Let's play until the end together ❤️ with 🧠
 ## ❓ FAQ <a name="faq"></a>
 - **Is there any restriction to adapting the DB to other porpuse??**
 
-  - No problem t.
+  - No problem your can go ahead.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [OPTIONAL] Better not to use the plural on the foreign key column instead, you can use a singular noun. example:

```sql
   CREATE TABLE visits (
   animal_id int,
   vet_id int,
   visit_date date
   );
```